tests/run_integration_tests.sh

```diff
@@ -150,6 +149,29 @@ while [  $i -lt ${#tests[@]} ]; do
             find output -type f -exec sed -i 's/e-00/e-0/g' {} \;
             find output -type f -exec sed -i 's/e+00/e+0/g' {} \;
         fi
+        
+        # some special handling for the *.errout files
+        for f in scripts/*.[Rr]ev ; do
+            tmp0=${f#scripts/}
+            tmp1=${tmp0%.[Rr]ev}
+            
+            # Delete all before the 1st occurrence of the string '   Processing file' (inclusive)
+            # Use a temporary intermediate file to make this work w/ both GNU and BSD sed
+            sed '1,/   Processing file/d' output/${tmp1}.errout > output/${tmp1}.errout.tmp
+            mv output/${tmp1}.errout.tmp output/${tmp1}.errout
+            
+            # Also delete the final line of failing tests, which reprints the path to the script
+            # that differs between Windows and Unix (has no effect if the line is absent)
+            sed '/   Error:\tProblem processing/d' output/${tmp1}.errout > output/${tmp1}.errout.tmp
+            mv output/${tmp1}.errout.tmp output/${tmp1}.errout
+            
+            # The dos2unix utility does not always catch OS-specific differences in path separators
```

Comment:
I think dos2unix only addresses line feeds by changing char(10) char(13) to char(10).

Reply:
Can you fix the comment?  Right now it sounds like dos2unix sometimes fixes pathnames, which it doesn't.